plaso/preprocessors/macos.py

```diff
@@ -34,16 +33,15 @@ def _FindKeys(self, key, names, matches):
     the matches list.
 
     Args:
-      key (plistlib._InternalDict): plist key.
+      key (dict): plist key.
```

Comment:
Do we know what is in the dict? e.g.
`dict[str, object]`

Reply:
Done